### PR TITLE
[DC-1019] Create test buckets and datasets in Python replacing bash scripts

### PR DESCRIPTION
**Pull request by lrwb-aou**
* uses python to create test buckets and datasets.

### CODE REVIEW

**Review by dcampbell-vumc — CHANGES_REQUESTED** [review]
Just a question about the unused `import os` statements in `create_datasets.py` and `create_buckets.py` and whether we can get rid of it or not.

File: data_steward/ci/create_buckets.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import os
```
[the remark this answers is not in the document]
Comment (lrwb-aou) [reply data_steward/ci/create_buckets.py:1]: nice catch!

File: data_steward/ci/create_datasets.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
+import os
```
[the remark this answers is not in the document]
Comment (lrwb-aou) [reply data_steward/ci/create_datasets.py:1]: good catch.

**Review by dcampbell-vumc — APPROVED** [review]
Changing the names of the modules to `test_*` and adding a module description clears up any confusion about what the modules are doing!

**Pull request merged**